ports/ttauri/portfile.cmake

```
@@ -0,0 +1,25 @@
+vcpkg_fail_port_install(ON_ARCH "x86" "arm" "arm64")
+vcpkg_fail_port_install(ON_TARGET "linux" "uwp" "osx")
+vcpkg_fail_port_install(ON_LIBRARY_LINKAGE "dynamic")
```

```suggestion
vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
```
This will perform some standard automatic conversions (dynamic lib + dynamic crt will convert to static lib + dynamic crt).